Components/Queues/Drivers/PhpAmqpLibDriver.php

```diff
@@ -149,8 +128,6 @@ public function createQueueMessage($queueName = null, $options = [])
     {
         $msg = new QueueMessage();
         $msg->setContext(new Context());
-        $this->setChannel();
-        $this->manager->declareQueue($queueName, AMQP_DURABLE, $options);
```

Comment:
Deleting this, you don't need anymore the `$queueName` and `$options` in the method definition

Reply:
Done.